src/WebSdk/Publish/Targets/CopyTargets/Microsoft.NET.Sdk.Publish.FilterFiles.targets

```diff
@@ -0,0 +1,37 @@
+﻿<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Publish.FilterFiles.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your web deploy projects from the command-line or the IDE.
+
+This file defines the steps in the standard package/publish process for collecting only files to run the web appliation.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--********************************************************************-->
+  <!-- Target _FilterIISAssets -->
+  <!--********************************************************************-->
+  <Target Name="_FilterIISAssets"
+          Condition="'$(PublishIISAssets)' == 'false'"
+          AfterTargets="ComputeResolvedFilesToPublishList"
+          BeforeTargets="CopyFilesToPublishDirectory">
+
+    <ItemGroup>
+      <ResolvedFileToPublish Update="@(ResolvedFileToPublish)"
+      Condition="'%(Filename)%(Extension)' == 'aspnetcorev2_inprocess.dll'">
```

Comment:
@jkotalik - Are there other IIS native dlls that I should filter?

Reply:
I thought about it but I wasn't sure if I would break somebody who relies on other metadata for these files. These seemed the least impactful.